osu.Game.Rulesets.Osu/OsuRuleset.cs

```diff
@@ -94,6 +94,7 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
                         new OsuModEasy(),
                         new OsuModNoFail(),
                         new MultiMod(new OsuModHalfTime(), new OsuModDaycore()),
+                        new OsuModSpunOut(),
```

Comment:
Couldn't this be regarded as automation as well?

Reply:
Moved intentionally as it is ranked and has a negative multiplier. All remaining automation mods are non-ranked, which is better defined IMO.